frontend/Dockerfile

```diff
@@ -15,22 +15,21 @@ RUN apt-get update && apt-get install -y make wget \
     libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 \
     libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 \
     libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 \
-    lsb-release xdg-utils nano vim procps \
+    lsb-release xdg-utils nano vim procps moreutils ripgrep \
     && rm -rf /var/lib/apt/lists/*
 
 COPY package*.json ./
-# the official node image provides an unprivileged user as a security best practice
-# but we have to manually enable it. We put it here so npm installs dependencies as the same
-# user who runs the app.
-# https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#non-root-user
-USER node
 RUN npm ci --verbose --no-optional && npm cache clean --force
 ENV PATH /opt/node_app/node_modules/.bin:$PATH
 
 # -- TODO, we should try turning this back on later.
 # ADD --chown=node . /corpora-frontend
 ADD . /corpora-frontend
 WORKDIR /corpora-frontend
+ADD ./src/configs/build.js src/configs/configs.js
+RUN mkdir -p node_modules
+RUN ln -sf /opt/node_app/node_modules/* /opt/node_app/node_modules/.bin ./node_modules/.
+RUN npm run build
```

Comment:
@jgadling we had to replace `next build` with this so that the `postbuild` step could run. Any concerns?

Reply:
Nope, I just didn't know what the best way to build the static app was!